backend/tests/unit/services/test_agent_registry.py

```
@@ -267,50 +220,43 @@
-        }
-        
-        registry = AgentRegistry(config=unicode_config)
-        
-        assert registry.get_agent_for_alert_type("AlertWithÜnicode") == "UnicodeAgent"
-        assert registry.get_agent_for_alert_type("Alert🚨Emergency") == "EmojiAgent"
+        }, [
+            ("AlertWithÜnicode", "UnicodeAgent"),
+            ("Alert🚨Emergency", "EmojiAgent")
+        ]),
+    ])
+    def test_registry_edge_cases(self, config_type, config, test_cases):
```

Ruff (0.12.2)

Unused method argument: `config_type`

(ARG002)